crates/signs/src/lib.rs

```diff
@@ -13,6 +13,8 @@ mod pole;
 const MAX_VISIBILITY_DISTANCE: f32 = 140.;
 const DISTANCE_FLAG_BIT: u32 = 0;
 
+const TIMER_BAR_ID: u32 = 2;
```

Comment:
I would rename `TIMER_BAR_ID` to `UPDATE_TIMER_FLAG_BIT` to so that the naming is consistent and more suggestive.

Reply:
Ok. Fwiw I took that scheme from `controller/lib.rs`, just didn't notice we already had related constants here.